src/sagemaker/fw_utils.py

```diff
@@ -613,7 +613,7 @@ def warn_if_parameter_server_with_multi_gpu(training_instance_type, distribution
 
 
     """
-    if training_instance_type == "local" or distributions is None:
+    if training_instance_type == "local" or distributions is None or "mpi" in distributions:
```

Comment:
I don't think you need to add mpi here - see ll. 624-626 with `ps_enabled`

Reply:
this itself is ll 624 right?

Reply:
Fair point
Got it!